### PR TITLE
v7.9.0

### PR DESCRIPTION
**Pull request by DanailH**

### CODE REVIEW

**Review by LukasTy — APPROVED** [review]
LGTM. 👍

**Review by arminmeh — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,70 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.9.0
+
+_Jul 5, 2024_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 📊 Add Heatmap charts
```
Comment (JCQuintas) [inline CHANGELOG.md:12]: We didn't release the package, so it might be better not to announce it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,70 @@
...
+#### `@mui/x-date-pickers-pro@7.9.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@7.9.0`.
+
+### Charts
+
+#### `@mui/x-charts@7.9.0`
+
+- [charts] Add Heatmap (#13209) @alexfauquette
+- [charts] Add initial `Zoom&Pan` to the Pro charts (#13405) @JCQuintas
```
Comment (JCQuintas) [inline CHANGELOG.md:47]: These two are actually on the `Pro` as well. I don't know if we should make a new section or hide these for now 😅

Comment (DanailH) [reply CHANGELOG.md:47]: If it's not released yet then we can postpone the new section.

File: CHANGELOG.md (hunk 1 above)
Comment (DanailH) [reply CHANGELOG.md:12]: ok, then in that case I'll remove it from the announcements section.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,70 @@
...
```
Comment (JCQuintas) [inline CHANGELOG.md:47]: Should we add something like `(unreleased)` to these?

```suggestion
- [charts] Add Heatmap (unreleased) (#13209) @alexfauquette
- [charts] Add initial `Zoom&Pan` to the Pro charts (unreleased) (#13405) @JCQuintas
```

**Review by cherniavskii — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,70 @@
...
+- 🌳 Add `selectItem` and `getItemDOMElement` methods to the TreeView component public API
+- ⛏️ Make the `usePickersTranslations` hook public in the pickers component
+- 🐞 Bugfixes
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.9.0`
+
```
Comment (KenanYusuf) [inline CHANGELOG.md:21]:
```suggestion

- [DataGrid] Add Skeleton loading overlay support (#13293) @KenanYusuf 

```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,70 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.9.0
+
+_Jul 5, 2024_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
```
Comment (KenanYusuf) [inline CHANGELOG.md:11]:
```suggestion

- 🔄 Add loading overlay variants, including a skeleton loader option to the Data Grid component. See [Loading overlay docs](https://mui.com/x/react-data-grid/overlays/#loading-overlay) for more details.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,70 @@
...
+### Tree View
+
+#### `@mui/x-tree-view@7.9.0`
+
+- [TreeView] Add `selectItem` and `getItemDOMElement` methods to the public API (#13485) @flaviendelangle
+
+### Docs
+
+- [docs] fix custom "no results overlay" demo in dark mode (#13715) @KenanYusuf
```
Comment (MBilalShafi) [inline CHANGELOG.md:62]:
```suggestion
- [docs] Fix custom "no results overlay" demo in dark mode (#13715) @KenanYusuf
```

**Review by KenanYusuf — APPROVED** [review]

**Review by MBilalShafi — APPROVED** [review]

**Pull request merged**